canvases/dark-editor-vue/src/components/Prompt.vue

```diff
@@ -30,6 +30,7 @@
               rows="1"
               placeholder="What are you building today?"
               class="w-full h-6 max-h-44 outline-none m-0 resize-none overflow-y-auto border-0 bg-transparent text-white py-0 pl-2 pr-11"
+              @keydown.enter.exact.prevent="submitPrompt"
```

Comment:
Here `exact` is to allow shift+enter to add a new line in the textarea. Should we make a comment about it or is it clear?

Reply:
I think it's fine - I suppose we'll all soon be fluent enough in Vue that this'll become common?